app/workers/articles/score_calc_worker.rb

```diff
@@ -0,0 +1,18 @@
+module Articles
+  class ScoreCalcWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :medium_priority
+
+    def perform(article_id)
+      article = Article.find_by(id: article_id)
+      return unless article
+
+      article.update_columns(score: article.reactions.sum(:points),
```

Comment:
I really like my jobs not to know things about the app's business logic, so how about moving this out to some dedicated object like `ArticleScoreUpdater` or even just `Article_#update_score`?

Reply:
Updated to `Article_#update_score`.